seeders/users.seeder.js

```diff
@@ -1,25 +1,34 @@
 import { Seeder } from "mongoose-data-seed";
 import User from "../src/models/User";
+import Instructor from "../src/models/Instructor";
 import { hashPassword } from "../src/utils/auth";
 
-const data = [
-  {
-    email: "instructor@example.com",
-    password: "instructor",
-    accessLevel: "instructor",
-  },
-  {
-    email: "admin@example.com",
-    password: "admin",
-    accessLevel: "admin",
-  },
-  {
-    email: "revoked@example.com",
-    password: "revoked",
-    accessLevel: "revoked",
-  },
-];
-
+const data = async () => {
+  const instructors = await Instructor.find({});
+  return [
+    {
+      email: "instructor@example.com",
+      password: "instructor",
+      accessLevel: "instructor",
+    },
+    {
+      email: "admin@example.com",
+      password: "admin",
+      accessLevel: "admin",
+    },
+    {
+      email: "revoked@example.com",
+      password: "revoked",
+      accessLevel: "revoked",
+    },
+    {
+      email: "test@example.com",
+      password: "password",
+      accessLevel: "instructor",
+      instructor: instructors[1],
```

Comment:
This will need to be reverted as well on this branch like it was on the query-files branch.

`git checkout origin/main seeders/users.seeder.js`